torch/utils/file_baton.py

```diff
@@ -39,9 +42,16 @@ def wait(self):
         The amount of time slept depends on the ``wait_seconds`` parameter
         passed to the constructor.
         """
+        tik = time.time()
         while os.path.exists(self.lock_file_path):
             time.sleep(self.wait_seconds)
 
+            # If lock file exists in the beginning and waited too long,
```

Comment:
> If lock file exists in the beginning

Hmm why does it matter whether the lock file already existed when this particular instance of FileBaton was initialized
If I'm using a file lock I'm most likely synchronizing different processes, and potentially doing try_acquire immediately after initializing anyway.